docs/how-to-setup-freecodecamp-locally.md

```diff
@@ -178,11 +178,11 @@ cp sample.env .env
 copy sample.env .env
 ```
 
-The keys in the `.env` file are *not* required to be changed to run the app locally. You can leave the default values copied over from `sample.env` as-is.
+The keys in the `.env` file is *not* required to be changed to run the app locally. You can leave the default values copied over from `sample.env` as-is.
```

Comment:
Please revert this line as the verb "are" is for the word "keys" and not "file".